msteams-platform/sbs-bots-with-sso.yml

```diff
@@ -139,7 +139,8 @@ items:
                    :::image type="content" source="~/assets/images/bots/newresourcecreation.png" alt-text="New resource creation":::
 
               1. If you've created a new resource group, select the required location from **New resource group location** dropdown list.
-        
+                 For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
Link to create resource group is not required here, Line 139 already covers this step.

Reply:
The link is removed as suggested.